tests/v2/test_waku_pagination.nim

```diff
@@ -0,0 +1,25 @@
+import
+  std/unittest,
+  ../../waku/node/v2/waku_types
+
+suite "pagination":
+  test "computeIndex: empty contentTopic test":
+      let
+          wm = WakuMessage(payload: @[byte 1, 2, 3])
+          index = wm.computeIndex()
+      check:
+          # the fields of the index should be non-empty
+          len(index.digest.data) != 0
+          index.receivedTime != 0
+
+  test "computeIndex: identical WakuMessages test":
+      let
+          wm = WakuMessage(payload: @[byte 1, 2, 3], contentTopic: "topic2")
+          index1 = wm.computeIndex()
+          wm2 = WakuMessage(payload: @[byte 1, 2, 3], contentTopic: "topic2")
+          index2 = wm2.computeIndex()
+
+      check:
+          # the digests of two identical WakuMessages must be the same
+          index1.digest == index2.digest
+
```

Comment:
@staheri14 this all should be 2 space indentation rather than 4. The mix won't work.

Reply:
Thanks! Addressed the indentation issues.